src/helpers.ts

```diff
@@ -0,0 +1,18 @@
+import { Person } from './types';
+
+export const getPeopleWithParents = (people: Person[]) => {
+  return people.map(person => {
+    const mother = people.find(mom => person.motherName === mom.name || null);
+    const father = people.find(dad => person.fatherName === dad.name || null);
```

Comment:
You don't need to manually return `null`, the `find` method will return `undefined` if it doesn't find anything:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find